src/isSentinelValue.js

```diff
@@ -0,0 +1,26 @@
+import { curryN } from 'ramda';
+
+import isInteger32 from './isInteger32';
+
+/**
+ * Checks whether the passed value is a sentinel value.
+ *
+ * @func isSentinelValue
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.33.0|v2.33.0}
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {boolean}
+ * @see {@link RA.isSentinelValue|isSentinelValue}
+ * @example
+ *
+ * RA.isSentinelValue(-1); //=> true
+ *
+ * RA.isSentinelValue('-1'); //=> false
+ * RA.isSentinelValue(1); //=> false
+ * RA.isSentinelValue([-1]); //=> false
+ */
+const isSentinelValue = curryN(1, (val) => isInteger32(val) && val === -1);
```

Comment:
Can we change this to idiomatic sentinel value check? The `isInteger` predicate already guarantees that value is signed 32 bit integer before it moves to `~val === 0`. If eslint is giving you issues, use `eslint-disable-line no-bitwise`.

```suggestion
const isSentinelValue = curryN(1, (val) => isInteger32(val) && ~val === 0);
```

Reply:
Is there any reason why you prefer `~val === 0` over `val === -1`. Isn't sentinel value directly defined as -1?

Reply:
Technically both are equivalent. The reason I prefer former (`~val === 0`) is that this operation actually performs a complement of the value and then we assert if the complement equals `+0` or `-0` and `0` respectively. This actually describe what the algorithm for computing sentinel value actually is rather then just asserting for `-1`.

I guess that's my reasoning about it.

Reply:
Addressed